tests/task_decorators/secured_task_test.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from builtins import *
+import http.client
+from unittest.case import TestCase
+
+from zsl.application.containers.web_container import WebContainer
+from zsl.application.modules.web.cors import CORSConfiguration
+from zsl.router.task import TaskConfiguration
+from zsl.task.task_decorator import json_input, secured_task
+from zsl.testing.db import IN_MEMORY_DB_SETTINGS
+from zsl.testing.http import HTTPTestCase
+from zsl.testing.zsl import ZslTestCase, ZslTestConfiguration
+from zsl.utils.security_helper import TOKEN_HASHED, TOKEN_RANDOM, TOKEN_SERVICE_SECURITY_CONFIG, compute_token
+
+
+class SecuredTask(object):
+
+    @json_input
+    @secured_task
+    def perform(self, data):
+        return "ok"
+
+
+class SecuredTaskTestCase(ZslTestCase, HTTPTestCase, TestCase):
+    CONFIG = IN_MEMORY_DB_SETTINGS.copy()
+    CONFIG.update({
+        'CORS': CORSConfiguration('origin'),
+        'TASKS': TaskConfiguration().create_namespace('test').add_routes(
+            {'secured_task': SecuredTask}).get_configuration(),
+        TOKEN_SERVICE_SECURITY_CONFIG: 'ahoj'
+    })
+
+    ZSL_TEST_CONFIGURATION = ZslTestConfiguration(
+        app_name='SecuredTaskTestCase', container=WebContainer,
```

Comment:
let's put every param on a single line and `)` on its own line, it could be more readable, please.